source/changelog/r115.inc

```diff
@@ -0,0 +1,122 @@
+* **[Preview]** |eFaktura| transactions with multiple identical Tax Groups will now be bundled together to produce a better visual experience.
+* **[Preview]** Fixed an issue where |eFaktura| transactions would not be generated in case the issuing Organization was missing phone number or contact email.
+
+
+Changed
+-------
+
+* Payment Import API has been slightly modified in terms of the output. The input is unchanged.
+* Simplified account configuration for |AvtaleGiro| and |eFaktura|. Multiple input fields has been removed (defining them will not break things, they will just be ignored).
+* Subscription Restarts will now be routed through the Order processing to facilitate handling of multiple Payment Agreement scenarios.
+* Multiple adjustments to the underlying Payment Import file processing, in order to improve feedback and resilience.
+* Scheduled Subscription Plan changes will no longer split the existing Subscription in cases where there is no need. For instance when changes are set to affect a future Subscription period.
+
+
+Deprecated
+----------
+We call out the following deprecations, the endpoints will be removed in a future version, the exact timing depending on existing use and adoption of replacements.
+
+* We are deprecating the endpoints ``/paymentdemand/{id}/credit/completely`` and ``/paymentdemand/{id}/credit/partially`` in favor of the afore mentioned ``/paymentdemand/{id}/credit`` endpoint.
+
+
+[Preview] New Feature : Email Domain based User Authorization
+-------------------------------------------------------------
+
+This release introduces building blocks for automating User Authorization based on the domain name of their email address.
+It is an alternative to :ref:`Site Access <_auth-site-access>` where all users created with a specific domain name, will be authorized to a given domain.
+
+It will work side-by-side with other user Authorization mapping concepts, so you can mix and match as you see fit.
+
+This is still in preview, and some of the UI elements are not yet in place. 
+Additionally some external modifications are needed in the IdP and a custom sign-up experience responsible for mapping the user to the domain authorization.
+
+Interested in participating in the preview? Contact support, find the details on {SUPPORTPAGE}.
+
+
+Merchant Client
+===============
+The following new features and improvements are now available in the merchant client.
+
+Added
+-----
+
+* **[Preview]** Support for listing the sum of not-invoice Billing Account transactions on the ledger summary.
+* **[Preview]** Support for configuring Domain Authorization on Subscriptions.
+* Merchants can now configure Self-Service behaviour for Subscription Cancellations.
+* Added an indicator showing if there are Invoices on a given subscriber.
+
+
+Fixed
+-----
+
+* An issue with the wrong value being show for Reminder Fees.
```

Comment:
It was the Tax amount in Invoices (just the overview) that were incorrect, for Reminders with Fees AND tax calculation for invoice fee